modules/tinymce/src/core/main/ts/api/html/Schema.ts

```diff
@@ -940,10 +940,10 @@ const Schema = (settings?: SchemaSettings): Schema => {
   /**
    * Returns a map with elements where white space is to be preserved. For example: <code>&#60;pre&#62;</code> or <code>&#60;script&#62;</code>.
    *
-   * @method getWhiteSpaceElements
+   * @method getWhitespaceElements
    * @return {Object} Name/value lookup map for white space elements.
    */
-  const getWhiteSpaceElements = Fun.constant(whiteSpaceElementsMap);
+  const getWhitespaceElements = Fun.constant(whiteSpaceElementsMap);
```

Comment:
Can you also please rename the `whiteSpaceElementsMap` to match this name please?